azure-pipelines-templates/build/job/dacpac-template.yml

```diff
@@ -0,0 +1,32 @@
+parameters:
```

Comment:
Not sure if this and some of the other YAML files are required for this repo?
I thought `das-payments-v2-common` would just contain Nuget packages for class libraries, and the database project would be in the `das-payments-v2-databases` repo

Reply:
Just to confirm, upon reviewing this file, it's just a template that was loaded in by DevOps when setting up the repo. For the purpose of this pull request, the file isn't in use so won't cause any issues but I'll drop a ticket on the backlog to have DevOps review and remove any unused files